uiServices/test/createConfigFile_test.go

```diff
@@ -0,0 +1,202 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"github.com/DATA-DOG/godog"
+	"github.com/DATA-DOG/godog/gherkin"
+	"io/ioutil"
+	"net/http"
+	"io"
+	"strings"
+	"os"
+	"github.com/Sirupsen/logrus"
+	"github.com/xtracdev/automated-perf-test/uiServices/src"
+	"encoding/xml"
+	"github.com/xtracdev/automated-perf-test/perfTestUtils"
+	"encoding/json"
+	"reflect"
+)
+
+type apiFeature struct {
+	resp   *http.Response
+	client *http.Client
+}
+
+func (a *apiFeature) resetResponse() {
+	a.client = &http.Client{}
+	a.resp = nil
+}
+
+func (a *apiFeature) iSendrequestTo(method, endpoint string) (err error) {
+	response, err := makeRequest(a.client, method, endpoint, "")
+	if err != nil {
+		return err
+	}
+	a.resp = response
+	return nil
+}
+
+func (a *apiFeature) theResponseCodeShouldBe(code int) error {
+
+	if code != a.resp.StatusCode {
+		return fmt.Errorf("expected response code to be: %d, but actual is: %d", code, a.resp.StatusCode)
+	}
+	return nil
+}
+
+//not currently needed for POST request. Will be needed for GET request in future sprints
+func (a *apiFeature) theResponseShouldMatchJSON(body *gherkin.DocString) (err error) {
+	return nil
+}
+
+func (a *apiFeature) theResponseBodyShouldBeEmpty() error {
+	defer a.resp.Body.Close()
+
+	bodyBytes, err := ioutil.ReadAll(a.resp.Body)
+
+	if err != nil {
+		logrus.Error(err)
+		return err
+	}
+
+	if len(bodyBytes) > 0 {
+		return errors.New("Body should be empty")
+	}
+	return nil
+}
+
+func theHeaderConfigsDirPathIs(path string) error{
+	expectedConfigsPathDir :=os.Getenv("GOPATH") + "/src/github.com/xtracdev/automated-perf-test/uiServices/test/GodogConfig.xml"
```

Comment:
for now this is fine, but we should add a requestHeaders map to the Feature and read from it to see what the header value is. Anywhere we reference `/uiServices/test/GodogConfig.xml` we should just read the request headers from `a *Feature`